### DISCUSSION TITLE
eorisis pixel plugin

### DISCUSSION THREAD

**Discussion by Giuseppe-TD** [akeeba/panopticon#850] [General]
i just want to inform that using the eorisis facebook pixel plugin for joomla breaks the panopticon plugin :D

**Comment by nikosdion** [comment]
You need to contact its developer, let them know their plugin breaks Joomla's API application, and point them to this article https://www.dionysopoulos.me/common-mistakes-writing-joomla-plugins.html 

Remember, we cannot do anything about badly written third party code. We know how Joomla is supposed to work, and how not to break it. Heck, I even [wrote a book about Joomla! software development](https://www.dionysopoulos.me/book.html). Other developers need to understand how Joomla! works, and not break it with their code :)

**Comment by Giuseppe-TD** [comment]
yes, i will do :D i just wanna inform the community of this problem :D
